apps/chat/src/store/models/models.epics.ts

```diff
@@ -236,14 +236,21 @@ const getInstalledModelIdsFailEpic: AppEpic = (action$, state$) =>
       const recentModelIds = ModelsSelectors.selectRecentModelsIds(
         state$.value,
       );
+      const availableModels = ModelsSelectors.selectModels(state$.value);
+      const firstAvailableModelId = availableModels[0].id;
```

Comment:
`availableModels[0]` can be undefined

Reply:
Thank you for the comment.
Added fallback.